### ISSUE TITLE
Month Year validation error.

### ISSUE THREAD

**Issue by ultimateakash** [JsDaddy/ngx-mask#572]
Hello, i tried to mask credit card expiry with ng mask.
```html 
<input mask="M0/00"  type="text"  formControlName="expiry" id="expiry" placeholder="mm/yy" />
```
but still user can enter invalid month. please help i want validation for month and year

MM/YY

**Comment by isysenko** [comment]
@ultimateakash Hi, will be fixed in the new mask version

**Comment by alandeg** [comment]
Hi @isysenko would you kindly confirm whether this bug has been fixed in version **8.1.7**?

![image](https://user-images.githubusercontent.com/9683932/71825566-6c67e100-309c-11ea-8c35-d68a9dab65c5.png)

The pattern I'm using here is _**0000-M0-d0**_
